core/src/main/scala/org/apache/spark/sql/SnappyParser.scala

```diff
@@ -1121,9 +1124,13 @@ class SnappyParser(session: SnappySession)
   }
 
   protected def dmlOperation: Rule1[LogicalPlan] = rule {
-    (INSERT ~ INTO | PUT ~ INTO) ~ tableIdentifier ~
-        ANY.* ~> ((r: TableIdentifier) => DMLExternalTable(r,
-        UnresolvedRelation(r), input.sliceString(0, input.length)))
+    INSERT ~ INTO ~ tableIdentifier ~ ANY.* ~> ((r: TableIdentifier) => DMLExternalTable(r,
```

Comment:
Indentation. INSERT and PUT can be aligned.

BTW one more style can be this: up to you...
protected def dmlOperation: Rule1[LogicalPlan] = rule {
    (INSERT ~ INTO ~ push(false) | PUT ~ INTO ~ push(true)) ~ tableIdentifier ~
        ANY.* ~> ((putInto: Any, r: TableIdentifier) =>
      if (putInto.asInstanceOf[Boolean]) {
      DMLExternalTable(r,
        UnresolvedRelation(r), input.sliceString(0, input.length))
      } else {
        // check type .... your code ...
        DMLExternalTable(r,
          UnresolvedRelation(r), input.sliceString(0, input.length))
      })
  }